lib/error/ParseError.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+class ParseError extends Error {
```

Comment:
Technically, it would have been better to call this "TransformError", as the word "Parse" is very strongly associated with JSON.parse. If somebody told me there was a "ParseError" I would think that we had sent invalid json, rather than that there was an error in the code that executes *after* the parse successfully completed. It might be a pain, but could you change this?

Reply:
Done